src/portal_visualization/builders/epic_builders.py

```diff
@@ -43,17 +53,102 @@ def apply(self):
     def _apply(self, conf):  # pragma: no cover
         pass
 
+    def zarr_store_url(self):
+        adata_url = self._build_assets_url(zarr_path, use_token=False)
+        return adata_url
+
+    def segmentations_url(self, img_path):
+        img_url = self._build_assets_url(img_path)
+        return (
+            img_url,
+            str(
+                re.sub(
+                    r"ome\.tiff?",
+                    "offsets.json",
+                    re.sub(IMAGE_PYRAMID_DIR, OFFSETS_DIR, img_url),
+                )
+            ),
+        )
 
-class SegmentationMaskBuilder(EPICConfBuilder):
 
+class SegmentationMaskBuilder(EPICConfBuilder):  # pragma: no cover
     def _apply(self, conf):
+        zarr_url = self.zarr_store_url()
         datasets = conf.get_datasets()
-        print(f"Found {len(datasets)} datasets")
-        # Proof of concept using one of the kaggle segmentation masks for now
-        # segmentations = ObsSegmentationsOmeTiffWrapper(
-        #     img_url='https://assets.hubmapconsortium.org/c9d9ab5c9ee9642b60dd351024968627/ometiff-pyramids/VAN0042-RK-3-18-registered-PAS-to-postAF-registered.ome_mask.ome.tif?token=AgndN7NVbn83wwDXjpnY1Y0lDoJj2j7zOGmn1WN6qr9pqdkjKmt9C1XYm4KrlWrOXE9rVJvpnEKrPjIXrlKd1hmDGjV',
-        #     # offsets_path=f'./{name}/{name}/offsets/{name}.segmentations.offsets.json',
-        #     obs_types_from_channel_names=True,
-        # )
-        # dataset.add_object(segmentations)
-        pass
+        # TODO: add the correct path to the segmentation mask ome-tiff (image-pyramid)
+        seg_path = f'{self.segmentations_url("seg")}/'
+        # print(seg_path)
+        seg_path = (
+            'https://assets.hubmapconsortium.org/c9d9ab5c9ee9642b60dd351024968627/'
+            'ometiff-pyramids/VAN0042-RK-3-18-registered-PAS-to-postAF-registered.ome_mask.ome.tif?'
+            'token=AgzQXm7nvOW32vWw0EPpKonwbOqjNBzNvvW1p15855NoYglJxyfkC8rlJJWy8V6E8MeyXOwlpKdNBnHb5qnv7f8oeeG'
```

Comment:
GitGuardian nagged me about including the token in the previously committed code, even though they expire quickly 😆 

I believe we can access the token via `self._groups_token` like the superclass does now that we're extending it: https://github.com/hubmapconsortium/portal-visualization/blob/main/src/portal_visualization/builders/base_builders.py#L106-L107